app/main.py

```diff
@@ -1,3 +1,47 @@
 class Distance:
-    # Write your code here
-    pass
+    def __init__(self, km: int):
+        self.km = km
+
+    def __str__(self):
+        return f"Distance: {self.km} kilometers."
+
+    def __repr__(self):
+        return f"Distance(km={self.km})"
+
+    def __add__(self, other):
+        if isinstance(other, Distance):
+            km = other.km
+        else:
+            km = other
+        return Distance(self.km + km)
+
+    def __iadd__(self, other):
+        if isinstance(other, Distance):
+            self.km += other.km
+        else:
+            self.km += other
+        return self
+
+    def __mul__(self, other):
+        return Distance(self.km * other)
+
+    def __truediv__(self, other):
+        return Distance(round((self.km / other), 2))
+
+    def __lt__(self, other):
+        return self.km < other
+
+    def __gt__(self, other):
```

Comment:
Consider that other can be a variable of different types (`int`, `float`, `Distance`)